HousingSearchApi.Tests/V1/UseCase/GetPersonListUseCase.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Threading.Tasks;
+using FluentValidation.Results;
+using HousingSearchApi.V1.Domain;
+using IPersonsGateway = HousingSearchApi.V1.UseCase.IPersonsGateway;
+
+namespace HousingSearchApi.Tests.V1.UseCase
+{
+    public class GetPersonListUseCase : IGetPersonListUseCase
```

Comment:
This looks like an implementation file, not a test file. Should it be here?

Reply:
When you say a test file?